cfpb/cf-component-demo
# Prevent cf-core from being compiled into docs.css

Fixing a bug experienced by @wernerc where old cf-core CSS was overriding new CSS on the cf-core docs.

## Changes

- Reverted importing of `cf-core.less` to be by `(reference)` so that vars and mixins may be used in `docs.less` without dumping the whole of cf-core into the compiled CSS.
- Reverted `.bowerrc`'s `directory` to `bower_components` so that `grunt bower` resumes functioning correctly.
- Updated Rainbow dependency because it's now a normal Bower package.

## Testing

1. Copy the updated code into your node_modules/cf-component-demo folder on cf-core
2. Run `grunt`
3. Serve the docs in a browser and observe the cf-core CSS no longer being loaded in at the top of `docs.css`

## Review

- @contolini 
- @mistergone

## Todos

- Manually add the vars and mixins from cf-core that we want to use in `docs.less` to the top of `docs.less`, so we can remove the `cf-core` dependency completely.

## Checklist
